app-development-wg/charter/README.md

```diff
@@ -0,0 +1,120 @@
+
+A core objective of this group is to identify and promote existing tools, standards, and practices within the CNCF ecosystem. By showcasing how these elements work together, we aim to map how existing tools and standards can enhance and complement practices and tools already familiar to developers and architects.
+
+In addition to identifying best practices, this group should champion open-source tools that promote seamless integration between development tools and adopt standards. These tools empower teams to construct custom interfaces tailored to their specific domains, ultimately reducing cognitive load and boosting developer productivity.
+
+The following list of goals will drive the working group agenda:
+
+- Identify and promote developer productivity tools within the cloud-native ecosystem. Raise awareness and facilitate the adoption of tools supporting and enhancing developer workflows in the cloud-native space.
+- Promote interoperability and standards across tools. Leverage shared APIs and SDKs across the industry to map the development ecosystem, document existing functionalities, and pinpoint areas for improvement.
+- Advocate for developer experience (DevEx) tools that streamline cloud-native development workflows.
+Assist with the organization of AppDeveloperCon, reviewing and proposing topics.
+- Identify how tools contribute to improving developer productivity metrics.
+- Provide early feedback and recommendations for projects and tools seeking CNCF adoption around cloud-native app development. 
+
+
+
+### Areas Out of Scope
+
+These are topics or initiatives that this WG will not target and work on. We might add new items or update this list if we find projects, standards, or initiatives this WG finds interesting to work on in future iterations. 
+- Create new standards.
+- Define how application development should be done.
+- Creation of a new CNCF open-source project.
+- Promote tools that are only useful for a specific stack/programming language.
+- Promote adopting security best practices throughout the development lifecycle, secure coding standards, and vulnerability scanning to mitigate risks and protect applications from potential threats.
+- Promote sustainability and efficiency in application development and tooling to optimize resource utilization and minimize environmental impact.
+
+
+### Deliverables to the TAG
+
+This working group aims to bring developers closer to the App Delivery TAG. Creating resources and collaborating on initiatives that matter to cloud-native developers and architects. Concrete initiatives and deliverables that this working group will concentrate on: 
+
+Complementing the CNCF landscape with a view targeted for cloud-native developers and architects
+Complementing the Platform WG Glossary for Application Development
+Complementing other working groups under the TAG App Delivery personas definitions
+Helping with the organization and selection of proposals for AppDeveloperCon
```

Comment:
Missing the bullets to render this as a list (see: https://github.com/cncf/tag-app-delivery/blob/b69babfe7897c50c6c2cfc3be88855e4965a1335/app-development-wg/charter/README.md)

```suggestion
* Complementing the CNCF landscape with a view targeted for cloud-native developers and architects
* Complementing the Platform WG Glossary for Application Development
* Complementing other working groups under the TAG App Delivery personas definitions
* Helping with the organization and selection of proposals for AppDeveloperCon
```